### DISCUSSION TITLE
Add shortcut alias

### DISCUSSION THREAD

**Discussion by AndreasWintherMoen** [dailydotdev/daily#1211] [Feature Requests]
The current shortcuts only use the URL as their name. This makes it less organized, particularly if several shortcuts have the same prefix such as in my screenshot below. For example, my CVPartner page and Sharepoint page get the same text and same favicon. 

![Screenshot 2024-06-10 at 09 38 22](https://github.com/dailydotdev/daily/assets/30579542/a0391044-076c-480b-a805-e5f77b7526c5)

A shortcut should have a name/alias which would be displayed instead. Creating new shortcuts would look more something like this:

<img src="https://github.com/dailydotdev/daily/assets/30579542/7b43d535-c607-4e8b-8dc0-6f8357f201d4" height="500px" />

**Comment by idoshamun** [comment]
Hi @AndreasWintherMoen,

Thanks a lot for the suggestion we will consider it
